src/state/process.rs

```diff
@@ -710,6 +690,11 @@ where
         )?;
         self.verify_address(&vote.voter, true)?;
```

Comment:
lack a critical check, you should check whether self.address is the correct leader of vote's epoch_id and round.

Reply:
and so in handle_signed_proposal

Reply:
This is handling signed vote, not handling aggregated vote. The critical check has occurred while handling signed proposal.

Reply:
yes, you are right.